### PR TITLE
Fixes the way we start and stop couchdb on bare metal CI agents

### PR DESCRIPTION
**Pull request by unlikelyzero**
### Describe your changes:
Fixes the way we start and stop couchdb on bare metal CI agents

Some drivebys:
- move everything to node18 for speed
- Add support for running our CI scripts on bare metal and playwright container
- bumper node/circle orb
- Generate code coverage and npm artifacts always
- Fix couchdb flake by running tests without parallelization

### All Submissions:

* [x] Have you followed the guidelines in our [Contributing document](https://github.com/nasa/openmct/blob/master/CONTRIBUTING.md)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/nasa/openmct/pulls) for the same update/change?
* [x] Is this change backwards compatible? For example, developers won't need to change how they are calling the API or how they've extended core plugins such as Tables or Plots.

### Author Checklist

* [ ] Changes address original issue?
* [x] Tests included and/or updated with changes?
* [x] Command line build passes?
* [x] Has this been smoke tested?
* [x] Testing instructions included in associated issue OR is this a dependency/testcase change?

### Reviewer Checklist

* [ ] Changes appear to address issue?
* [ ] Reviewer has tested changes by following the provided instructions?
* [ ] Changes appear not to be breaking changes?
* [ ] Appropriate automated tests included?
* [ ] Code style and in-line documentation are appropriate?
* [ ] Has associated issue been labelled unverified? (only applicable if this PR closes the issue)
* [ ] Has associated issue been labelled bug? (only applicable if this PR is for a bug fix)

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -23,9 +27,8 @@ commands:
       - restore_cache_cmd:
           node-version: << parameters.node-version >>
       - node/install:
-          install-npm: true
           node-version: << parameters.node-version >>
-      - run: npm install --prefer-offline --no-audit --progress=false
+      - run: npm install --no-audit --progress=false
```
Comment (unlikelyzero) [inline .circleci/config.yml:31]: this is causing the machine's cache to conflict with the container's

File: .circleci/config.yml [hunk 2]
```diff
@@ -69,7 +72,7 @@ commands:
     - run: npm run cov:e2e:report || true
     - run: npm run cov:e2e:<<parameters.suite>>:publish
 orbs:
-  node: circleci/node@4.9.0
+  node: circleci/node@5.1.0
```
Comment (unlikelyzero) [inline .circleci/config.yml:75]: this new version upgrades to latest available nvm version _before_ attempting an install

File: .circleci/config.yml [hunk 3]
```diff
@@ -110,7 +113,11 @@ jobs:
           path: dist/reports/tests/
       - store_artifacts:
           path: coverage
-      - generate_and_store_version_and_filesystem_artifacts
+      - when:
+          condition:
+            equal: [ 42, 42 ] # Always generate version artifacts regardless of test failure https://discuss.circleci.com/t/make-custom-command-run-always-with-when-always/38957/2
+          steps:
+            - generate_and_store_version_and_filesystem_artifacts
```
Comment (unlikelyzero) [inline .circleci/config.yml:120]: always generate artifacts -- even on failure

File: .circleci/config.yml [hunk 4]
```diff
@@ -234,7 +268,7 @@ workflows:
       - visual-test:
           node-version: lts/gallium
       - e2e-couchdb:
-          node-version: lts/gallium
+          node-version: lts/hydrogen
```
Comment (unlikelyzero) [inline .circleci/config.yml:271]: cache conflict during testing

File: .circleci/config.yml [hunk 5]
```diff
@@ -37,24 +40,24 @@ commands:
...
           paths:
             - ~/.npm
             - node_modules
   generate_and_store_version_and_filesystem_artifacts:
     description: "Track important packages and files"
     steps:
       - run: |
-          mkdir /tmp/artifacts
-          printenv NODE_ENV >> /tmp/artifacts/NODE_ENV.txt
+          [[ $EUID -ne 0 ]] && (sudo mkdir -p /tmp/artifacts && sudo chmod 777 /tmp/artifacts) || (mkdir -p /tmp/artifacts && chmod 777 /tmp/artifacts)
```
Comment (unlikelyzero) [inline .circleci/config.yml:59]: sudo not available in container

File: .circleci/config.yml (hunk 4 above)
Comment (unlikelyzero) [reply .circleci/config.yml:275]: Update: Just moved them all to 18 so we could get npm dep resolution speed improvements

**Review by ozyx — APPROVED** [review]
LGTM

**Pull request merged**